cmd/call-api-endpoint/main.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"strings"
+
+	"github.com/alecthomas/kong"
+
+	"github.com/edulinq/autograder/internal/api"
+	"github.com/edulinq/autograder/internal/api/core"
+	"github.com/edulinq/autograder/internal/cmd"
+	"github.com/edulinq/autograder/internal/config"
+	"github.com/edulinq/autograder/internal/log"
+)
+
+var args struct {
+	config.ConfigArgs
+	cmd.CommonOptions
+
+	Endpoint   string   `help:"Endpoint of the desired API." arg:""`
+	Parameters []string `help:"Parameter(s) for the endpoint" arg:"" optional:""`
+}
+
+func main() {
+	kong.Parse(&args,
+		kong.Description("Execute an API request to the specified endpoint. For more information on available API endpoints, see the API resource file at: ../resources/api.json"),
+	)
+
+	err := config.HandleConfigArgs(args.ConfigArgs)
+	if err != nil {
+		log.Fatal("Failed to load config options.", err)
+	}
+
+	var description *core.EndpointDescription
+
+	apiDescription := api.Describe(*api.GetRoutes())
+	for endpoint, requestResponse := range apiDescription.Endpoints {
+		if endpoint == args.Endpoint {
+			description = &requestResponse
+			break
+		}
+	}
+
+	if description == nil {
+		log.Fatal("Failed to find the endpoint.", log.NewAttr("endpoint", args.Endpoint))
+
+		// Return after log.Fatal() sets the exit code to 1 to avoid proceeding during tests.
```

Comment:
This is a case where you can have a better comment with less description.
By specifying "1" specifically, you are committing to keeping this comment up-to-date in case the default exit status for a log.Fatal() changes.
That would be a super hard thing to do (searching for "1" would not be very helpful).